docs/source/markdown/podman-service.1.md

```diff
@@ -0,0 +1,47 @@
+% podman-search(1)
+
+## NAME
+podman\-service - Run an API service
+
+## SYNOPSIS
+**podman service** [*options*]
+
+## DESCRIPTION
+The **podman service** command creates a listening service that will answer API calls for podman.  You may
+optionally provide an endpoint for the API in URI form.  For example, *unix://tmp/foobar.sock* or *tcp:localhost:8080*.
+If no endpoint is provided, defaults will be used.  The default endpoint for a rootfull
+service is *unix:/run/podman/podman.sock* and rootless is *unix:/$XDG_DIR/podman/podman.sock* (for
```

Comment:
XDG_DIR or XDG_RUNTIME_DIR?

Reply:
i think i have it correct ... anyone else?

Reply:
@baude Tom is correct. XDG_RUNTIME_DIR